content/200-concepts/100-components/02-prisma-client/053-client-extensions/120-query.mdx

```diff
@@ -129,6 +136,25 @@ const xprisma = prisma.$extends({
 })
 ```
 
+### Modify all Prisma Client operations
+
+Use the `$allOperations` method to modify all query methods present in Prisma Client. The `$allOperations` can be used on both model operations and raw queries.
+
+You can modify all methods as follows:
+
+```ts
+const xprisma = prisma.$extends({
+  query: {
+    $allOperations({ model, operation, args, query }) {
+      // modify all operations
+      return query(args)
+    },
+  },
+})
+```
+
+In the event a raw query is invoked, the `model` argument passed to the callback will be `null`.
```

Comment:
What will be `operation` in this case?

Reply:
`operation` will be the method that is invoked by Prisma Client, e.g. `$queryRaw`, or `findMany` etc

Reply:
`model` would be `undefined`, not `null`.

Reply:
`query` is the callback to execute it